src/Saml2/Provider.php

```diff
@@ -736,4 +731,10 @@ protected function mapUserToObject(array $user)
     {
         throw new NotSupportedException();
     }
+
+    protected function cacheKey(string $key): string
+    {
+        $hash = md5($this->getConfig('acs') ?: $this->getConfig('metadata'));
+        return "socialite_saml2_{$hash}_{$key}";
```

Comment:
could we use `sprintf` here please?

Reply:
Done, and fixed the other issue the linter found